app/chrony/README.md

```diff
@@ -2,21 +2,19 @@
 
 Basic chrony NTP client monitoring.
 
-* [Template App chrony active](Template_App_chrony_active.xml)
-
 This template is part of [RaBe's Zabbix template and helpers
 collection](https://github.com/radiorabe/rabe-zabbix).
 
-## Template
-
-### Items
-
-* Memory usage (rss) of "chronyd" processes (proc.mem[chronyd,chrony,,,rss])
-* Number of "chronyd" processes (proc.num[chronyd,chrony])
+## Template App chrony active
 
+### Items 
+* Memory usage ($5) of "$1" processes (`proc.mem[chronyd,chrony,,,rss]`)
```

Comment:
Ugly solutions like https://stackoverflow.com/a/3067130 exist for xslt-1.0. I'll probably end up refactoring this to something dom based to gain enough flexibility to implement this.